### ISSUE TITLE
Native Scrolling

### ISSUE THREAD

**Issue by cortner** [aquamacs-emacs/aquamacs-emacs#98]
This is really not an issue but a feature request, I hope it is ok to post here? 

What is the chance to get native scrolling (i.e. by pixel rather than by line) in Aquamacs in a future version. I have seen it work fairly well in 
   https://github.com/railwaycat/emacs-mac-port

There are many things I love about Aquamacs and keep it as my main editor, but I find the way it scrolls very straining compared to say Sublime.

**Comment by davidswelt** [comment]
I agree.  We will get this feature from the experimental branch that you are referring to, although there is no timeline just yet.

## 

http://aquamacs.org -- Aquamacs: Emacs on Mac OS X
http://aquamacs.org/donate -- Could we help you? Return the favor and support the Aquamacs Project!

**Comment by kennyluck** [comment]
Hmm... Does anyone have a pointer to some user experience study justifying this request? I've not noticed this "jagged scrolling" ever since I use Aquamacs (so that's more than 4 years, what a shame) and isn't it a better user experience to make any line "always clear".

Yeah, now I think of this. This does feel kind of laggy. I shouldn't have read this :(

**Comment by x-u** [comment]
Switching from Sublime&Evernote to Emacs/Org-Mode is one of the most painful experience. Though I do have hope that I will eventually be very familiar and productive with Emacs, the horrible experience without Smooth Scrolling still frustrates me. The experience itself is like having used a phone with Retina display for many years and one suddenly has to switch back to phones with the old crappy display. 

Please read about this post which also mentioned scrolling issue: http://emacs.stackexchange.com/questions/271/what-is-the-difference-between-aquamacs-and-other-mac-versions-of-emacs
I'm sure many new Emacs users on Mac have to face this dilemma which builds to adopt.

**Comment by cortner** [comment]
The [emacs-mac-port](https://github.com/railwaycat/emacs-mac-port) works so well for me that, for the past few months I have not used Aquamacs. I miss the convenience of having everything set up for me though - it was a fair bit of work to set it up to behave roughly like Aquamacs. But for me the smooth scrolling just has a huge (positive) impact on my stress level.

I hope Aquamacs will introduce this soon, then I'll probably return to it.

**Comment by jlinde** [comment]
It would be really nice if we could get this supported.
